fab/operations/db.py

```diff
@@ -42,6 +42,15 @@ def ensure_preindex_completion():
 
 
 @roles(ROLES_DB_ONLY)
+def ensure_checkpoints_safe():
+    extras = '--print-only' if env.force else ''
+    with cd(env.code_root):
+        sudo('{env.virtualenv_root}/bin/python manage.py validate_kafka_pillow_checkpoints {extras}'.format(
+            env=env, extras=extras
+        ))
```

Comment:
This doesn't tell people they can override with `--force=true`.  Also thinking that people might not know what the error messages mean or what to do about them. Guessing that's probably OK since it's a pretty rare occurrence but some docs somewhere could be good.